test/tools/ossfuzz/lpsolver/FuzzerSolverInterface.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+	This file is part of solidity.
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+
+#include <test/tools/ossfuzz/lpsolver/FuzzerSolverInterface.h>
+
+#include <range/v3/view/enumerate.hpp>
+
+using namespace solidity::test::fuzzer::lpsolver;
+using namespace solidity::util;
+using namespace std;
+
+FuzzerSolverInterface::FuzzerSolverInterface(bool _supportModels):
+	m_solver(_supportModels)
+{
+	m_solvingState.variableNames.emplace_back("");
+}
+
+void FuzzerSolverInterface::addLEConstraint(LinearExpression _lhs)
+{
+	// Move constant to RHS
+	if (_lhs[0])
+		_lhs[0] = -_lhs[0];
```

Comment:
I mean this is also not really needed in your case, is it?

Reply:
Right. This was a semantic convention to know how to read a fuzzer input while debugging i.e.,

```
1, 23, 2, 1
```

would be read as `2*x1 + x2 = 23` instead of `2*x1 + x2 = -23` i.e., not having to negate fuzzer generated input to derive the constraint.

Reply:
Fixed it by adjusting `linearExpression` to follow the first convention. Gets rid of the intermediate functions.